less/global.less

```diff
@@ -43,4 +43,9 @@ img {
   max-width: 800px;
   width: 100%;
   margin: 0 auto;
+
+  @media @mobile {
+    width: 95%;
+  }
```

Comment:
Good job on adding this to your mobile view! You usually want to avoid text sitting on the edge of the screen.